test/game/test_shop.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from balatro_gym.cards.voucher import ALL_VOUCHERS
+from balatro_gym.game.shop import Shop, ShopState
+
+
+@pytest.mark.unit
+def test_generate_shop_state() -> None:
+    shop = Shop()
+    state = shop.generate_shop_state(1)
+    voucher = state.vouchers[0]
+    assert isinstance(state, ShopState)
+    state = shop.generate_shop_state(2)
```

Comment:
I think there is another case to check which is to check that e.g. on round 2, assert that the shop doesn't have vouchers for purchase IF the voucher has already been purchased that round.